android/app/src/main/java/com/ecomap/ecomap/clients/ecomap/http/ApiClient.kt

```diff
@@ -79,26 +115,193 @@ object ApiClient {
         errorListener: ErrorListener
     ): JsonObjectRequest {
         val requestPayload = JSONObject()
-        requestPayload.put("firstName", firstName)
-        requestPayload.put("lastName", lastName)
-        requestPayload.put("username", username)
-        requestPayload.put("password", password)
+
+        requestPayload.put(FIELD_NAME_FIRST_NAME, firstName)
+        requestPayload.put(FIELD_NAME_LAST_NAME, lastName)
+        requestPayload.put(FIELD_NAME_USERNAME, username)
+        requestPayload.put(FIELD_NAME_PASSWORD, password)
 
         return JsonObjectRequest(
-            Request.Method.POST, getApiUrl("/users"), requestPayload,
+            Request.Method.POST, URL_USERS, requestPayload,
+            { response -> listener.onResponse(mapUser(response)) },
+            errorListener
+        )
+    }
+
+    /**
+     * Returns the containers with the specified filter.
+     * @param containerCategory Container category to filter by.
+     * @param limit Amount of resources to get for the provided filter.
+     * @param offset Amount of resources to skip for the provided filter.
+     * @param listener Volley response listener.
+     * @param errorListener Volley response error listener.
+     * @return Volley request.
+     */
+    fun listContainers(
+        containerCategory: ContainerCategory? = null,
+        limit: Int = 100,
+        offset: Int = 0,
+        token: String,
+        listener: Listener<ContainersPaginated>,
+        errorListener: ErrorListener
+    ): JsonObjectRequest {
+        var url = URL_CONTAINERS +
+                "?$FIELD_NAME_PAGINATION_LIMIT=$limit" +
+                "&$FIELD_NAME_PAGINATION_OFFSET=$offset"
+        if (containerCategory != null) {
+            url += "&$FIELD_CONTAINER_CATEGORY=${mapDomainContainerCategory(containerCategory)}"
+        }
+
+        return object : JsonObjectRequest(
+            Method.GET, url, null,
             { response ->
+                val containers = ArrayList<Container>(limit)
```

Comment:
```suggestion
                val containers = arrayListOf<Container>(limit)
```
It's the same thing but it's more common to see it used in Kotlin with this syntax

Reply:
The behavior is not the same between the two approaches.
Refs: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/array-list-of.html